src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,51 @@
+@import "../utils/variables";
+
+.nav__list {
+  display: flex;
+  height: 100%;
+  list-style: none;
+}
+
+.nav__items {
+  height: 100%;
+
+  &:not(:last-child) {
+    margin-right: 20px;
+  }
+}
+
+.nav__link {
+  position: relative;
+  display: block;
+  height: 100%;
+  padding: 23px 0;
+  text-transform: uppercase;
+  font-size: 12px;
+  font-weight: 500;
+  color: $main-accent;
+  text-decoration: none;
+}
+
+.is-active {
+  color: $header-accent;
+  position: relative;
+}
+
+.collor_one:hover {
```

Comment:
The class name 'collor_one' is not following the BEM naming convention. Consider changing it to 'nav__link--hover' or another appropriate name.